lib/timed_semaphore.rb

```diff
@@ -8,17 +8,17 @@
 # @author ssuljic
 #
 # @example
-#   threads = []
-#   semaphore = TimedSemaphore.new(2, 3)
+# threads = []
```

Comment:
Embarrassing but I actually don't know how. This is technically my first pull request!